examples/siro_sandbox/app_states/app_state_fetch.py

```diff
@@ -55,72 +61,122 @@ def on_environment_reset(self, episode_recorder_dict):
         ]
 
         self._nav_helper.on_environment_reset()
+        self._pick_helper.on_environment_reset()
+        self._pick_helper.agent_feet_height = self._get_agent_feet_height()
```

Comment:
We may potentially pass `self._get_agent_feet_height()` to `self._pick_helper.on_environment_reset()` and set `agent_feet_height` there.

Reply:
Addressed in new PR